dashboard/app/views/shared/_concept_trophy_block.html.haml

```diff
@@ -2,10 +2,6 @@
   %h4 #{t('nav.popup.mastery')}
 
   %table{style: 'max-width: 290px;'}
-    - if concept_progress
-      - concept_progress.each_pair do |concept, counts|
-      - completion = counts[:current].to_f / counts[:max]
-        = render partial: 'shared/concept_trophy_row', locals: {concept: concept, completion: completion}
-    - else
-      - Concept.cached.each do |concept|
-        = render partial: 'shared/concept_trophy_row', locals: {concept: concept, completion: 0}
+    - Concept.cached.each do |concept|
+      - completion = (concept_progress || {})[concept.name] || 0
```

Comment:
The partial is expecting a number between 0 and 1, but this line sets it to an object like `{obj: 'loop_while', current: 0, max: 0}`.